web_app_drawer/static/src/less/variables.less

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+// App Drawer / Icons
+
+@app-drawer-icon-size:      5.625em;
+@app-drawer-icon-margin:    0.625em;
+@app-drawer-width:          80%;
+@app-drawer-title-height:   2.9em;
+
+// Navbar
+
+@navbar-height:  2.3em;
+@navbar-padding-vertical: .565em;
+@navbar-padding-horizontal: .565em;
```

Comment:
Not sure if you meant creating a widget for putting the icons in, or creating a widget to put something in the blank space remaining...

In 1st case, maybe a reusable Qweb template would be enough... I don't think we want to put that menu everywhere. In 2nd, I think the goal of filling all blank space possible is not really a good one. App menu should be just for apps, no more, no less (IMHO). You don't reach perfection when you have nothing more to add, but when you have nothing more to remove :wink:.